tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -453,13 +453,13 @@ def print_perf_counters(perf_counts_list):
                 f"{str(pi.status):<20} "
                 f"layerType: {pi.node_type[:max_print_length - 4] + '...' if (len(pi.node_type) >= max_print_length) else pi.node_type:<20} "
                 f"execType: {pi.exec_type:<20} "
-                f"realTime (ms): {pi.real_time / timedelta(milliseconds=1):<10.3f} "
-                f"cpuTime (ms): {pi.cpu_time / timedelta(milliseconds=1):<10.3f}")
+                f"realTime (ms): {pi.real_time.microseconds:<10} "
+                f"cpuTime (ms): {pi.cpu_time.microseconds:<10}")
 
             total_time += pi.real_time
             total_time_cpu += pi.cpu_time
-        print(f'Total time:     {total_time / timedelta(milliseconds=1)} milliseconds')
-        print(f'Total CPU time: {total_time_cpu / timedelta(milliseconds=1)} milliseconds\n')
+        print(f'Total time:     {total_time.microseconds} milliseconds')
+        print(f'Total CPU time: {total_time_cpu.microseconds} milliseconds\n')
```

Comment:
I'm not sure I understand the problem statement.
> Legacy API report performance counters in us, new API reports them in ms. Add unit convertion in legacy convert utils.

Does benchmark_app use legacy API?

Anyway `print(f'Total CPU time: {total_time_cpu.microseconds} milliseconds\n')` looks ridiculous when you extract `microseconds` field of `total_time_cpu` and report it as `milliseconds`. You should probably change milliseconds->microseconds and ms->us. Same for C++.